### PR TITLE
LPS-121998 Add new service Impl methods to account for the new Country fields

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke  @pei-jung I know I have several regens and bchan wants them consolidated but for review, I left them separated out.

https://issues.liferay.com/browse/LPS-121998

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:reopen

**Comment by drewbrokke** [comment]
ci:test:relevant

File: modules/apps/address/address-test/src/testIntegration/java/com/liferay/address/service/test/CountryLocalServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		Assert.assertEquals(updatedCountry.getNumber(), country.getNumber());
+		Assert.assertEquals(
+			updatedCountry.getPosition(), country.getPosition(), 0);
+		Assert.assertEquals(
+			updatedCountry.isShippingAllowed(), country.isSubjectToVAT());
+		Assert.assertEquals(
+			updatedCountry.isSubjectToVAT(), country.isSubjectToVAT());
+	}
+
+	private Country _addCountry() throws Exception {
```
Comment (drewbrokke) [inline modules/apps/address/address-test/src/testIntegration/java/com/liferay/address/service/test/CountryLocalServiceTest.java:106]: I'd just inline this for now

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@alee8888 a couple more action items:

- [ ] Add methods to the remote service layer
- [ ] Deprecate read methods that don't take a `companyId`
- [ ] In the deprecated methods, use the new finders that take a `companyId` and pass in the default `companyId`.
- [ ] If it's reasonable at this stage, replace usages of the deprecated methods.

**Pull request closed without merging**